### ISSUE TITLE
Did the unused part of physical transaction log get really intact until the logical part is full?

### ISSUE THREAD

**Issue by yassine1616** [MicrosoftDocs/feedback#1210]
**🛑 IMPORTANT**: You can get your feedback addressed faster if you **use the comment section for the article in which you encountered a problem**.

**Link to article:**
https://docs.microsoft.com/en-us/sql/relational-databases/sql-server-transaction-log-architecture-and-management-guide?view=sql-server-2017.

**Problem:**
My question is about "Transaction Log Physical Architecture" part.
What doses indicated in this article is "New log records are added at the end of the logical log and expand toward the end of the physical log". 
However, what does illustrated as figure makes sense : New log records expand toward the end of the physical log BUT not touch the unused part of VLF since it may use the truncated part in the start file.
In other words, in the expansion operation of the logical file, the truncated part from the start of the physical file is a priority to unused part in the end of the physical file.

It is worth noting that you indicated two figures of the same file with different VLF numbers 4 and 5, which makes confusion.

Thanks for reading.

**Comment by rothja** [comment]
@yassine1616 Hi. I assume you are talking about these diagrams:
![image](https://user-images.githubusercontent.com/1580454/53974704-23887a00-40d1-11e9-8d27-63472ed73fc7.png)

I do see that we are missing VLF 5 in the second diagram. I believe if we added that to the second diagram (and add it filled up before wrapping around to the beginning, that this would fix the problem you're seeing? Let me know if that is the case.

**Comment by yassine1616** [comment]
Hi,
First of all thank you. Your reply is very helpful.
Indeed the missed part what confused me.
